submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                required
+              />
+              <span class="input_name input_email_title"></span>
+              <input
+                type="password"
+                name=""
+                class="form-sign_pay_method_inputs input_password"
+                pattern="(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*"
+                title="Minimum 8 characters password,should have UPPERCASE, lowercase and Number"
+                required
+              />
+              <span class="input_name input_password_title"></span>
+            </div>
+            <div class="wrapper_form-choose_paysystem">
+              <input
+                type="radio"
+                name="payment"
+                id="visa_mastercard"
+                class="visually__hidden radio_input_payment"
+              />
+              <label for="visa_mastercard" class="form-choose_payment_radio">
+                <img
+                  src="images/mastercard.svg"
+                  alt="mastercards_icon"
+                  class="icon_paymethod"
+                />
+                <img
+                  src="images/visa.svg"
+                  alt="visa_icon"
+                  class="icon_paymethod"
+                />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="paypal"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="paypal"
+                class="form-choose_payment_radio paypal_label"
+              >
+                <img src="images/paypal.svg" alt="" class="icon_paypal" />
+              </label>
+              <input
+                type="radio"
+                name="payment"
+                id="amazon"
+                class="visually__hidden radio_input_payment"
+              />
+              <label
+                for="amazon"
+                class="form-choose_payment_radio amazon_label"
+              >
+                <img
+                  src="images/amazon.svg"
+                  alt="amazon icon"
+                  class="icon_amazon"
+                />
+              </label>
+            </div>
+            <div class="wrapper_form-card_number">
+              <input
+                type="text"
+                class="form-sign_pay_method_inputs  input_card_number validate_on_focus"
+                placeholder="0000 0000 0000 0000"
+                minlength="16"
+                maxlength="16"
+              />
+              <span class="input_card_data_title card_number_title"></span>
+              <input
+                type="datetime"
+                class=" form-sign_pay_method_inputs  input_date validate_on_focus"
+                placeholder="DD/MM/YY"
+                maxlength="6"
+                minlength="6"
+              />
+              <span class="input_card_data_title input_date_title"></span>
```

Comment:
This text must be a label. Also, it must be able to edit him from HTML.

Reply:
Done.